src/adagrams.js

```diff
@@ -1,8 +1,150 @@
 const Adagrams = {
   drawLetters() {
-    // Implement this method for wave 1
+    const allLetters = {
+  //pool of available letters and the qty they each have
```

Comment:
I really like that you store the letter frequencies and then build the pool on the fly - this is much easier to read than a giant array full of 9 'A's, 2 'B's, etc.